client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-detail-main/account-detail-main.component.ts

```diff
@@ -13,6 +14,7 @@ export class AccountDetailMainComponent extends BaseModelRequestHandlerComponent
         if (params[`id`] !== oldParams[`id`]) {
             const id = +params[`id`];
             this.subscribeTo(getAccountDetailSubscriptionConfig(id), { hideWhenDestroyed: true });
+            this.subscribeTo(getMeetingListSubscriptionConfig(), { hideWhenDestroyed: true });
```

Comment:
This subscription is potentially quite big therefore `hideWhenDestroyed` should not be added to prevent reopening this multiple times. Also it would be better to open this subscription only when the add to meeting page was opened. 

You could also add a new subscription config that subscribes less fields and does not subscribe to archived meetings.